index.js

```diff
@@ -465,6 +465,18 @@ class Runtime {
     this.classFactory.choose(specifier, callbacks);
   }
 
+  chooseSync (specifier) {
+    const instances = [];
+    this.choose(specifier, {
+      onMatch (c) {
+        instances.push(c);
+      },
+      onComplete () {
+      }
+    });
+    return instances;
+  }
```

Comment:
This assumes that the underlying implementation is always synchronous. It unfortunately isn't in the case of Dalvik, and we may choose to make the others asynchronous later. So it would be better to return a `Promise` in `choose()` so make it easy to use with `await`.